rhobs/alerting/data_plane/prometheus.pipeline_alerts.yaml

```diff
@@ -49,7 +49,7 @@ spec:
             runbook_url: https://gitlab.cee.redhat.com/konflux/docs/sop/-/blob/main/pipeline-service/slos/execution-overhead.md
         - alert: CorePipelineControllerRepeatedRestarts
           expr: |
-            sum by (source_cluster) (increase(kube_pod_container_status_restarts_total{namespace="openshift-pipelines", pod=~"tekton-.*"}[5m])) > 0
+            sum by (source_cluster) (increase(kube_pod_container_status_restarts_total{namespace="openshift-pipelines", pod=~"tekton-.*|pipelines-as-code-.*"}[5m])) > 0
```

Comment:
I tried the query `kube_pod_container_status_restarts_total{namespace="openshift-pipelines", pod=~"tekton-.*|pipelines-as-code-.*"}` in the prod-rh01 OCP metrics console and it looks correct @divyansh42 

Good job figuring out the regex.